File: JavaScript Solutions/1041.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');

let Y = parseFloat(lines.shift()),
    X = parseFloat(lines.shift());

if (X > 0 && Y > 0)
    console.log("Q1");

else if (X > 0 && Y < 0)
    console.log("Q2");
    
else if(X < 0 && Y < 0)
    console.log("Q3");
    
else if (X < 0 && Y > 0)
    console.log("Q4");
    
else if (X === 0 && Y === 0)
    console.log("Origem");

else if (X===0)
    console.log("Eixo X");

else
    console.log("Eixo Y");

File: Python Solutions/1041.py
dados= input().split()

eixoY= float(dados[0])
eixoX= float(dados[1])

if eixoY == 0:
    if eixoX != 0:
        print("Eixo Y")
    
    else:
        print("Origem")

elif eixoX > 0:
    if eixoY > 0:
        print("Q1")
    else:
        print("Q2")

elif eixoX < 0:
    if eixoY > 0:
        print("Q4")
    else:
        print("Q3")
        
else:
    print("Eixo X")
